lib/pure/times.nim

```diff
@@ -262,11 +285,26 @@ proc `==`*(a, b: Time): bool {.
     rtl, extern: "ntEqTime", tags: [], raises: [], noSideEffect, borrow.}
   ## Returns true if ``a == b``, that is if both times represent the same point in time.
 
+proc `-`*(a, b: EpochDate): int32 {.borrow.}
```

Comment:
I think this should return the `Duration` type from #6978

Reply:
As soon as Duration will appear in devel I can do that, otherwise it is problematic to test